packages/launch/src/focused-launch/index.tsx

```diff
@@ -27,14 +27,25 @@ const FocusedLaunch: React.FunctionComponent = () => {
 
 	const { enablePersistentSuccessView } = useDispatch( LAUNCH_STORE );
 
+	// Force Success view to be the default view when opening Focused Launch modal.
+	// This is used in case the user opens the Focused Launch modal after launching
+	// the site (e.g. when redirected back after the checkout screen)
 	React.useEffect( () => {
 		if ( isSiteLaunched ) {
 			enablePersistentSuccessView();
 		}
+	}, [ isSiteLaunched, enablePersistentSuccessView ] );
+
+	// This class is used to hide the "Launch" button from the block editor's header
+	// @TODO:
+	//   - if this is a block editor specific feature, move to the LaunchContext
+	//     and only specify when including Focused Launch thorugh Editing Toolkit
+	//   - think about a less hacky way to achieve this (e.g. @wordpress/hooks?)
```

Comment:
Good catch! Let's move this to Editing Toolkit for now since this side effect is applied only inside the editor (hiding the Launch button).

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/47808/commits/1a7836f19e721ce5f257c2270f26acff89f08220

Reply:
Wonder if we should still add a janitorial issue to implement this functionality in a less hacky way? Adding a class to the `body` works for now but it's not very robuts

Reply:
Here it is 👉  https://github.com/Automattic/wp-calypso/issues/47988
It's not assigned to any milestone since this more a general [editor - launch] integration issue.